common/components/register_model/spec.yaml

```diff
@@ -52,12 +53,17 @@ inputs:
     optional: false
     description: Path to the model directory
 
-  model_info_path:
+  download_details:
     type: uri_file
     optional: true
-    description: A JSON file to provide model registration specific settings (name, type, description, tags, properties). This will take priority over values provided through component inputs
+    description: A JSON file which contains information related to model download.
+
+  model_metadata:
+    type: uri_file
+    optional: true
+    description: YAML file that contains model metadata confirming to Model V2 [contract](https://azuremlschemas.azureedge.net/latest/model.schema.json)
 
```

Comment:
As suggested by @ManojBableshwar @elesel @vizhur,
please add tags like below: 

```
tags:
  Preview: ""
```

This will show like this on UI: 

![image](https://user-images.githubusercontent.com/61145377/227761273-ae56b0ad-c938-4178-8087-28f50f349ec1.png)

Reply:
Added